packages/runtime/test/consumption/requests.test.ts

```diff
@@ -96,6 +96,7 @@ describe("Requests", () => {
             expect(sLocalRequest.status).toBe(LocalRequestStatus.Draft);
             expect(sLocalRequest.content.items).toHaveLength(1);
             expect(sLocalRequest.content.items[0]["@type"]).toBe("TestRequestItem");
+            if (sLocalRequest.content.items[0]["@type"] !== "TestRequestItem") throw new Error("Unexpected type.");
```

Comment:
Same as above.

Reply:
If you remove this line, you will get an error. This is because of TypeScript's inability to refine types based on assertions like `expect`.